models/espace-agent/chiffre-affaires/index.ts

```diff
@@ -16,7 +16,7 @@ export const getChiffreAffairesProtected = async (
   return clientApiEntrepriseChiffreAffaires(siret).catch((error) =>
     handleApiEntrepriseError(error, {
       siret,
-      apiResource: 'ChiffreAffairesDGFiP',
+      apiResource: 'getChiffreAffairesProtected',
```

Comment:
Not sure about this as this is not a name of a resource. Maybe just `ChiffreAffairesProtected`?

Reply:
Done !